project_purchase_utilities/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2018 Gontzal Gomez - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "Project Purchase Utilities",
+    "version": "12.0.1.1.0",
+    "license": "AGPL-3",
+    "depends": [
+        "project",
+        "purchase",
+        "hr_timesheet",
+    ],
+    "author": "AvanzOSC",
```

Comment:
@amcor you forgot to add OCA as author (I did the same mistake) :stuck_out_tongue:

Reply:
done!